extensions/httpserver/description.yml

```
@@ -0,0 +1,30 @@
+extension:
+  name: httpserver
+  description: Simple DuckDB HTTP API extension
+  version: 1.0.0
+  language: SQL & C++
+  build: cmake
+  license: MIT
```

```suggestion
  license: MIT
  excluded_platforms: "wasm_mvp;wasm_eh;wasm_threads"
```
Unsure what you are using underneath, but I assume this is not there on Wasm. I think it's preferable to just not have it there than having it broken.

---

This is a limit on wasm not running tests (yet).

---

Thanks Carlo you're 100% right I doubt the WASM version would be able to open sockets anyway. Perhaps we can think of a websocket based approach if that's an option later on in time.